swift/apple/FirezoneNetworkExtension/PacketTunnelProvider.swift

```diff
@@ -305,29 +304,3 @@ class PacketTunnelProvider: NEPacketTunnelProvider {
     completionHandler(data)
   }
 }
-
-extension NEProviderStopReason: @retroactive CustomStringConvertible {
```

Comment:
The last developer added these, and we don't use them. We will be showing more actionable errors if we need to communicate these cases to the user.